src/components/List/index.tsx

```diff
@@ -50,19 +70,30 @@ export const StyledListItem = styled(ListItem)<ListItemProps>`
 
 const StyledListSubItem = styled(ListItem)<ListItemProps>`
   &.MuiButtonBase-root.MuiListItem-root {
-    margin: 4px 0;
+    margin: 4px 0 4px 15px;
+    width: calc(100% - 15px);
```

Comment:
Can we give a variable name to this `15px` ? Not just to understand what it means but because I think it also affects the `margin` property of this element as well as the `::before` pseudoselector.

Reply:
Took a second look at the design, it seems 10px is more fitting. I wouldn't extract it to a variable since its all in the same place and doesn't really have a meaning.